hack/dockerfiles/test.buildkit.Dockerfile

```diff
@@ -0,0 +1,236 @@
+# syntax = tonistiigi/dockerfile:runmount20180925
+
+ARG RUNC_VERSION=00dc70017d222b178a002ed30e9321b12647af2d
+ARG CONTAINERD_VERSION=v1.1.3
+# containerd v1.0 for integration tests
+ARG CONTAINERD10_VERSION=v1.0.3
+# available targets: buildkitd, buildkitd.oci_only, buildkitd.containerd_only
+ARG BUILDKIT_TARGET=buildkitd
+ARG REGISTRY_VERSION=2.6
+ARG ROOTLESSKIT_VERSION=20b0fc24b305b031a61ef1a1ca456aadafaf5e77
+
+# available targets: buildkitd, buildkitd.oci_only, buildkitd.containerd_only
+ARG BUILDKIT_TARGET=buildkitd
+
+# git stage is used for checking out remote repository sources
+FROM --platform=$BUILDPLATFORM alpine AS git
+RUN apk add --no-cache git
+
+# xgo is a helper for golang cross-compilation
+FROM --platform=$BUILDPLATFORM tonistiigi/xx:golang AS xgo
```

Comment:
where's dockerfile for xx?

Reply:
https://github.com/tonistiigi/xx/blob/master/golang/Dockerfile